spec/javascripts/tests/EmbeddedGoogleAnalytics_spec.js

```diff
@@ -0,0 +1,64 @@
+describe.only('EmbeddedGoogleAnalytics', function() {
+  'use strict';
+
+  var sandbox;
+
+  before(function(done) {
+    var self = this;
+
+    requirejs([
+      'jquery',
+      'EmbeddedGoogleAnalytics'
+    ],
+    function(
+      $,
+      EmbeddedGoogleAnalytics
+    ) {
+      self.$html = $(window.__html__['spec/javascripts/fixtures/EmbeddedGoogleAnalytics.html']).appendTo('body');
+      self.$fixture = self.$html.find('[data-dough-component="EmbeddedGoogleAnalytics"]');
+      self.EmbeddedGoogleAnalytics = EmbeddedGoogleAnalytics;
+      sandbox = sinon.sandbox.create();
+
+      done();
+
+    }, done);
+  });
+
+  after(function() {
+    this.$html.remove();
+    sandbox.restore();
+  });
+
+  describe('after initialised', function() {
+    before(function(done) {
+      this.component = new this.EmbeddedGoogleAnalytics(this.$fixture, this.$fixture.data('doughEmbeddedGoogleAnalyticsConfig'));
+      this.component.init();
+      this.component.$el.show();
+      done();
+    });
+
+    it('renders auth link', function() {
+      expect(this.component.$el.find("#embed-api-auth-container:contains('Access Google Analytics')")).should.not.be.empty;
```

Comment:
I think you can use `to.not.be.empty` here, sort of blending should and expect assertions otherwise